setupgui/windows/odbcdialogparams.rc

```diff
@@ -221,30 +221,35 @@ IDD_TAB4 DIALOGEX 0, 0, 307, 245
-    RTEXT           "IAM IDP ARN:", IDC_STATIC, 3, 108, 58, 18
-    EDITTEXT        IDC_EDIT_IAM_IDP_ARN, 65, 107, 136, 12, ES_AUTOHSCROLL
-    LTEXT           "IDP Port:", IDC_STATIC, 207, 125, 36, 10
-    EDITTEXT        IDC_EDIT_IDP_PORT, 243, 124, 51, 12, ES_AUTOHSCROLL | ES_NUMBER
-    RTEXT           "AWS Region:", IDC_STATIC, 3, 126, 58, 18
-    EDITTEXT        IDC_EDIT_AUTH_REGION, 65, 125, 136, 12, ES_AUTOHSCROLL
-    RTEXT           "IAM Host:", IDC_STATIC, 3, 145, 58, 18
-    EDITTEXT        IDC_EDIT_AUTH_HOST, 65, 144, 136, 12, ES_AUTOHSCROLL
-    LTEXT           "IAM Port:", IDC_STATIC, 207, 144, 36, 18
-    EDITTEXT        IDC_EDIT_AUTH_PORT, 244, 143, 51, 12, ES_AUTOHSCROLL | ES_NUMBER
-    RTEXT           "IAM Expire Time:", IDC_STATIC, 3, 163, 58, 18
-    EDITTEXT        IDC_EDIT_AUTH_EXPIRATION, 65, 162, 136, 12, ES_AUTOHSCROLL | ES_NUMBER
+    LTEXT           "Federated Authentication Mode:",IDC_STATIC,207,7,80,18
+    COMBOBOX        IDC_EDIT_FED_AUTH_MODE,207,27,79,12,CBS_DROPDOWN | CBS_AUTOHSCROLL | CBS_SORT | WS_VSCROLL | WS_TABSTOP
+    RTEXT           "IDP Username:",IDC_STATIC,4,6,58,18
+    EDITTEXT        IDC_EDIT_IDP_USERNAME,65,6,136,12,ES_AUTOHSCROLL
+    RTEXT           "IDP Password:",IDC_STATIC,4,27,58,18
+    EDITTEXT        IDC_EDIT_IDP_PASSWORD,65,27,136,12,ES_PASSWORD | ES_AUTOHSCROLL
+    RTEXT           "IDP Endpoint:",IDC_STATIC,4,47,58,18
+    EDITTEXT        IDC_EDIT_IDP_ENDPOINT,65,46,136,12,ES_AUTOHSCROLL
+    RTEXT           "App ID:",IDC_STATIC,4,67,58,18
+    EDITTEXT        IDC_EDIT_APP_ID,65,67,136,12,ES_AUTOHSCROLL
+    RTEXT           "IAM Role ARN:",IDC_STATIC,3,88,58,18
+    EDITTEXT        IDC_EDIT_IAM_ROLE_ARN,65,87,136,12,ES_AUTOHSCROLL
+    RTEXT           "IAM IDP ARN:",IDC_STATIC,3,108,58,18
+    EDITTEXT        IDC_EDIT_IAM_IDP_ARN,65,107,136,12,ES_AUTOHSCROLL
+    LTEXT           "IDP Port:",IDC_STATIC,207,125,36,10
+    EDITTEXT        IDC_EDIT_IDP_PORT,243,124,51,12,ES_AUTOHSCROLL | ES_NUMBER
+    RTEXT           "AWS Region:",IDC_STATIC,3,126,58,18
+    EDITTEXT        IDC_EDIT_AUTH_REGION,65,125,136,12,ES_AUTOHSCROLL
+    RTEXT           "IAM Host:",IDC_STATIC,3,145,58,18
+    EDITTEXT        IDC_EDIT_AUTH_HOST,65,144,136,12,ES_AUTOHSCROLL
+    LTEXT           "IAM Port:",IDC_STATIC,207,144,36,18
+    EDITTEXT        IDC_EDIT_AUTH_PORT,244,143,51,12,ES_AUTOHSCROLL | ES_NUMBER
+    RTEXT           "IAM Expire Time:",IDC_STATIC,3,163,58,18
+    EDITTEXT        IDC_EDIT_AUTH_EXPIRATION,65,162,136,12,ES_AUTOHSCROLL | ES_NUMBER
+    LTEXT           "Client Connect Timeout:",IDC_STATIC,207,47,86,10
+    EDITTEXT        IDC_EDIT_CLIENT_CONNECT_TIMEOUT,207,59,51,12,ES_AUTOHSCROLL | ES_NUMBER
+    LTEXT           "Client Socket Timeout:",IDC_STATIC,207,76,75,10
+    EDITTEXT        IDC_EDIT_CLIENT_SOCKET_TIMEOUT,207,88,51,12,ES_AUTOHSCROLL | ES_NUMBER
+    CONTROL         "&Enable SSL",IDC_CHECK_ENABLE_SSL,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,207,108,47,10

```

Comment:
No expert on resource files, but is the `&` supposed to be there for Enable SSL? Also, image of the changes please

Reply:
Yes